addons/web_editor/static/lib/odoo-editor/src/commands/deleteForward.js

```diff
@@ -40,7 +43,9 @@ HTMLElement.prototype.oDeleteForward = function (offset) {
         return;
     }
     const firstOutNode = findNode(
-        rightLeafOnlyPath(...(firstInlineNode ? rightPos(firstInlineNode) : [this, offset])),
+        rightLeafOnlyPathNotBlockNotEditablePath(
```

Comment:
So if I get this right, the only change in this commit is that it uses `rightLeafOnlyNotBlockNotEditablePath` instead of `rightLeafOnlyPath` and `rightLeafOnlyNotBlockPath`? Which means we ignore blocks and not-editable node, correct? I would have expected this to _prevent_ deleting a block, rather than actually allowing it so I'm surprised.

Reply:
It prevent the deletion _inside_ the `contenteditable=false` but it must delete the node that is `contenteditable=false`.

Reply:
:+1: